### PR TITLE
Feature/domicile transfer select children

### PR DESCRIPTION
**Pull request by VilliGunn**
# \<Description\>

https://app.asana.com/0/1199733748699731/1199513406635467

## What

- Second screen of children domicile transfer.
- Using hardcoded values in checkboxes
- Extended one shared component `CheckboxController` to be able to utilize the `large` property
  - Also changed how tooltips were being displayed.

## Screenshots / Gifs

Screenshot 2021-01-12 at 15 48 35

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/shared/form-fields/src/lib/CheckboxController/CheckboxController.tsx [hunk 1]
```diff
@@ -73,12 +76,8 @@ export const CheckboxController: FC<CheckboxControllerProps> = ({
                   label={option.label}
                   value={option.value}
                   hasError={error !== undefined}
+                  tooltip={option.tooltip}
                 />
-                {option.tooltip && (
```
Comment (Rikkiabb) [inline libs/shared/form-fields/src/lib/CheckboxController/CheckboxController.tsx:77]: Why are we removing this?

**Review by Rikkiabb — APPROVED** [review]
One question, looks good 🚢

**Review by baering — APPROVED** [review]
LGTM

**Review by madebynoam — APPROVED** [review]
Lgtm!

**Comment by Rikkiabb** [comment]
:shipit:

**Review by Rikkiabb — APPROVED** [review]
🚢

**Pull request merged**